MockKit/MockPumpManager.swift

```diff
@@ -60,6 +63,19 @@ public final class MockPumpManager: TestingPumpManager {
         return MockPumpManager.pumpReservoirCapacity
     }
 
+
+    public func roundToSupportedBasalRate(unitsPerHour: Double) -> Double {
+        return supportedBasalRates.filter({$0 <= unitsPerHour}).max() ?? 0
+    }
+
+    public func roundToSupportedBolusVolume(units: Double) -> Double {
+        return supportedBolusVolumes.filter({$0 <= units}).max() ?? 0
+    }
+
+    public var supportedBolusVolumes: [Double] {
+        return supportedBasalRates
```

Comment:
This seems odd, but I guess it is a mock pump manager.

Reply:
It's pretty close to truth for actual pumps.